sphinx_gallery/gen_gallery.py

```diff
@@ -340,6 +340,33 @@ def call_memory(func):
     gallery_conf["compress_images"] = compress_images
     gallery_conf["compress_images_args"] = compress_images_args
 
+    # deal with matplotlib_animations
+    animations = gallery_conf["matplotlib_animations"]
+    if not isinstance(animations, (tuple, list)):
+        # Original option may be only a boolean.
+        animations = (animations,)
+    if len(animations) not in (1, 2):
```

Comment:
Is this too confusing ?
```suggestion
    if len(animations) not in (1, 2) and isinstance(enabled := animations[0], bool):
```
This error message is informative for the error below as well, so we can save some config checking.